yuniql-core/Source.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Yuniql.Core
+{
+    /// <summary>
+    /// Information source for yuniql configuration values
+    /// </summary>
+    public enum Source
+    {
+        /// <summary>
+        /// Default value assigned to configuration parameter
+        /// </summary>
+        Default,
```

Comment:
For consistency, lets name all constants and enums with ALLCAPS and no underscore.

Reply:
You mean ALLCAPS with underscore like this _ENVIRONMENT_VARIABLE_ or like this _ENVIRONMENTVARIABLE_ ?